### ISSUE TITLE
Error "Invalid type" when we call the setUserId function

### ISSUE THREAD

**Issue by SquirrelMobile** [Pushwoosh/pushwoosh-appcelerator-titanium#47]
When i set `pushwoosh.setUserId("1234");` , i have this error :

``````
[ERROR]  message = "Invalid type passed to function";
[ERROR]     nativeLocation = "-[ComPushwooshModuleModule setUserId:]ComPushwooshModuleModule.m:242)";
[ERROR]     nativeReason = "expected: String, was: Array";
``````

in this code : https://github.com/Pushwoosh/pushwoosh-appcelerator-titanium/blob/master/Module-Source/ios/Classes/ComPushwooshModuleModule.m#L242 , i think `
ENSURE_TYPE(args, NSString);` should be `ENSURE_TYPE(args[0], NSString);` ?

**Comment by tomgiessel** [comment]
When can we expect a new release that fixes this issue? I've seen you've fixed the issue with https://github.com/Pushwoosh/pushwoosh-appcelerator-titanium/commit/db18680f296a91d053f783cbcec719be058b9ae0.

**Comment by wfhm** [comment]
@SquirrelMobile @tomgiessel the release is expected to be out during this week.

**Comment by wfhm** [comment]
@SquirrelMobile @tomgiessel fixed in 3.4.0
